scripts/cml-autostart/autostart.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+ensure_dir() {
+    local install_dir="$1"
+    if ! test -d "$install_dir"; then
+        echo "Error: $install_dir isn't there"
+        exit 1
+    fi
+}
+
+install_default() {
+    local install_dir="/etc/default"
+    ensure_dir $install_dir
+    cat >"$install_dir/$(basename cml-autostart)" <<EOF
+USERNAME="admin"
+PASSWORD="password-that-needs-to-be-changed"
+TITLE_REGEX="\(autostart\)"
+EOF
+    chown root:root "$install_dir/$(basename cml-autostart)"
+    chmod 500 "$install_dir/$(basename cml-autostart)"
```

Comment:
Does the defaults file need to be executable?  Maybe 0600 would be better, so this file is easier to edit by sysadmin.